mqlc/builtin_resource.go

```diff
@@ -153,6 +154,9 @@ func compileResourceWhere(c *compiler, typ types.Type, ref uint64, id string, ca
 	if err != nil {
 		return types.Nil, err
 	}
+	if refs.isStandalone {
+		log.Warn().Msg("Detected a standalone function in a where-clause! Please make sure you use filter the list, as this is most likely unintended")
```

Comment:
"use filter the list" doesn't make sense to me. :) Just "filter the list" or "use a filter on the list"?

Reply:
```suggestion
		log.Warn().Msg("Detected a standalone function in a WHERE clause! This is probably unintended. Be sure to filter the list.")
```